### ISSUE TITLE
Adjusting what gets shown on the event page when event is live

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/WebsiteOne#810]
UI/UX design issue

**Comment by tansaku** [comment]
![current layout](https://www.dropbox.com/s/81zophkn2vxf997/Screenshot%202016-09-09%2016.26.00.png?dl=1)

**Comment by tansaku** [comment]
I think that the join now should be a button of similar design and location to the start hangout button

**Comment by mattwr18** [comment]
Hey @tansaku I don't have this orange button on my AgileVentures account, neither did we find it anywhere in the code base. 
We were able to find a blue `Start Hangout Instructions` button next to the `Start Jitsi Meet` button. 
Is the idea to make the JOIN NOW button similar to this `btn-primary` button? Or are we missing where this `Start a Hangout On Air` is coming from?
![Screenshot at 2018-08-09 07:59:44.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5b0ffc32a47b6b0012fec340/125516c66e82c728ace21e0d46db978826878dba87e6ab03a549b0c679177f3e795cee7b29c5b37fe517225b5b5b5cb053181d1ba6acda6fb4bc2e758c0e09efd8685b27c53518aaba8a1cbf01ee228a48d3b0ae965e82b96371b8d6d6056b5c17129bfe5a3fa19c48cb50f543e34e5ca21b3071ab.png)

**Comment by tansaku** [comment]
hi @mattwr18 so the orange button used to be provided by google.  They discontinued the "start a hangout on air" button so we replaced it with the blue one.  

So yes I think that "join now" could look similar to the "start" buttons by adding the btn-primary style - and perhaps moving the position - when I opened the issue I was thinking about how we use the screen real-estate.

To be honest I'm just not sure what's the best design any more  ... :-)
